diff --git a/node_modules/ansi-styles/index.js b/node_modules/ansi-styles/index.js
index 5d82581a..064dc398 100644
--- a/node_modules/ansi-styles/index.js
+++ b/node_modules/ansi-styles/index.js
@@ -157,7 +157,4 @@ function assembleStyles() {
 }
 
 // Make the export immutable
-Object.defineProperty(module, 'exports', {
-	enumerable: true,
-	get: assembleStyles
-});
+module.exports = assembleStyles();
diff --git a/node_modules/bl/bl.js b/node_modules/bl/bl.js
index 40228f87..08bfd744 100644
--- a/node_modules/bl/bl.js
+++ b/node_modules/bl/bl.js
@@ -1,6 +1,6 @@
 'use strict'
 
-const DuplexStream = require('readable-stream').Duplex
+const DuplexStream = require('readable-stream/lib/_stream_duplex')
 const inherits = require('inherits')
 const BufferList = require('./BufferList')
 
diff --git a/node_modules/clone/clone.js b/node_modules/clone/clone.js
index ba200c2f..bf33d370 100644
--- a/node_modules/clone/clone.js
+++ b/node_modules/clone/clone.js
@@ -1,4 +1,3 @@
-var clone = (function() {
 'use strict';
 
 /**
@@ -158,9 +157,5 @@ function __getRegExpFlags(re) {
 };
 clone.__getRegExpFlags = __getRegExpFlags;
 
-return clone;
-})();
 
-if (typeof module === 'object' && module.exports) {
   module.exports = clone;
-}
diff --git a/node_modules/execa/index.js b/node_modules/execa/index.js
index aad9ac88..2823ca2e 100644
--- a/node_modules/execa/index.js
+++ b/node_modules/execa/index.js
@@ -1,7 +1,10 @@
 'use strict';
 const path = require('path');
 const childProcess = require('child_process');
-const crossSpawn = require('cross-spawn');
+const crossSpawn = {
+  _parse: require('cross-spawn/lib/parse'),
+  _enoent: require('cross-spawn/lib/enoent')
+};
 const stripEof = require('strip-eof');
 const npmRunPath = require('npm-run-path');
 const isStream = require('is-stream');
diff --git a/node_modules/readable-stream/lib/_stream_readable.js b/node_modules/readable-stream/lib/_stream_readable.js
index 192d4514..b235dbd7 100644
--- a/node_modules/readable-stream/lib/_stream_readable.js
+++ b/node_modules/readable-stream/lib/_stream_readable.js
@@ -39,7 +39,7 @@ var EElistenerCount = function EElistenerCount(emitter, type) {
 /*<replacement>*/
 
 
-var Stream = require('./internal/streams/stream');
+var Stream = require('stream');
 /*</replacement>*/
 
 
diff --git a/node_modules/readable-stream/lib/_stream_writable.js b/node_modules/readable-stream/lib/_stream_writable.js
index a2634d7c..3997422a 100644
--- a/node_modules/readable-stream/lib/_stream_writable.js
+++ b/node_modules/readable-stream/lib/_stream_writable.js
@@ -57,13 +57,13 @@ Writable.WritableState = WritableState;
 /*<replacement>*/
 
 var internalUtil = {
-  deprecate: require('util-deprecate')
+  deprecate: require('util').deprecate
 };
 /*</replacement>*/
 
 /*<replacement>*/
 
-var Stream = require('./internal/streams/stream');
+var Stream = require('stream');
 /*</replacement>*/
 
 
